repo/packages/K/kafka/2/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "kafka",
+  "version": "0.9.3.0",
+  "scm": "https://github.com/mesos/kafka",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Kafka running on top of Apache Mesos",
+  "framework": true,
+  "tags": ["message", "broker", "messaging"],
+  "images": {
+    "icon-small": "https://d1vubr0evspla.cloudfront.net/img/icon-small.png",
+    "icon-medium": "https://d1vubr0evspla.cloudfront.net/img/icon-medium.png",
+    "icon-large": "https://d1vubr0evspla.cloudfront.net/img/icon-large.png",
+    "screenshots": ["https://d1vubr0evspla.cloudfront.net/img/screen-0-kafka.png"]
+  },
+  "preInstallNotes": "This will install Apache Kafka DCOS Service.",
```

Comment:
The purpose of the pre install notes is to provide the user with information to make an install decision.  We typically have resource requirements here... like number of nodes, cpu and mem requirements.